src/readers/XMLParser.cpp

```diff
@@ -1,35 +1,22 @@
 #include <algorithm>    //  std::replace
 
 #include "XMLParser.hpp"
-#include "XMLUtils.hpp"
 #include "utils.hpp"
 
 namespace RedatamLib
 {
-using std::cerr, std::endl;
+using std::cerr;
+using std::endl;
 
-XMLParser::XMLParser()
+// Constructor
```

Comment:
@litalbarkai hi! after reading pugixml docs, I am 50% sure that we don't need a constructor/destructor, but I can be wrong

Reply:
@pachadotdev I think you're right :)
these function definitions should be deleted and the .hpp files should have "= default" in the Ctor and Dtor declarations